.github/workflows/maven.yml

```diff
@@ -35,18 +35,20 @@ jobs:
        
     steps:
     - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
-      with:
-        persist-credentials: false
+      with:
+        persist-credentials: false

```

Comment:
What are the changes here?

Reply:
I don't know why git thinks there's a change here, I didn't edit these lines.

Reply:
This looks like a line endings issue. I have updated the repo to normalize the line endings and use a `.gitattributes` file. See [gitattributes](https://www.git-scm.com/docs/gitattributes).

If you rebase on master this should fix all issues.

Reply:
> This looks like a line endings issue. I have updated the repo to normalize the line endings and use a `.gitattributes` file. See [gitattributes](https://www.git-scm.com/docs/gitattributes).
> 
> If you rebase on master this should fix all issues.

@aherbert 
Should all Commons repos use this feature?

Reply:
Quite a few of them already do. Both Sebb and I collectively have added it to many of the repos. E.g. it is present in lang, numbers, statistics, math, geometry, now logging. It is not in text or imaging. It is very easy to add but caution must be applied if the repo has binary files. These can be added as exceptions to the attribute file. Imaging has binary image files that may require an exclusion (the auto detection is quite good).